### PR TITLE
Print finding stack trace before libFuzzer details

### PR DESCRIPTION
**Pull request by bertschneider**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	"scripts": {
 		"prepare": "husky install",
 		"build": "tsc -b tsconfig.build.json",
-		"clean": "git clean -dfX",
+		"clean": "rimraf -g **/node_modules **/tests/**/package-lock.json **/examples/**/package-lock.json **/dist **/coverage packages/fuzzer/build packages/fuzzer/prebuilds",
```
Comment (zgtm) [inline package.json:22]: thx! :+1:

**Review by zgtm — APPROVED** [review]
Looks good!

We should definitely do something about these flaky tests!

File: packages/core/core.ts [hunk 2]
```diff
@@ -314,13 +312,23 @@ export function asFindingAwareFuzzFn(
 	originalFuzzFn: fuzzer.FuzzTarget,
 	dumpCrashingInput = true,
 ): FindingAwareFuzzTarget {
+	function printAndDump(error: unknown): void {
+		cleanErrorStack(error);
+		if (
+			!(error instanceof FuzzerSignalFinding) ||
```
Comment (br-lewis) [inline packages/core/core.ts:318]: What would happen when `error` is an instance of `FuzzerSignalHandling`?

Comment (bertschneider) [reply packages/core/core.ts:318]: Only `FuzzerSignalHandling` errors with `exitCode` `Ok` should not be printed and dumped, all others should. Change the check?

Comment (br-lewis) [reply packages/core/core.ts:318]: Yeah I think changing it to `!(error instanceof FuzzerSignalFinding && error.exitCode === FuzzingExitCode.Ok)` would make it easier to understand.

Comment (bertschneider) [reply packages/core/core.ts]: Done.

**Pull request merged**